entity-types/ext-nvidia_jetson/definition.yml

```diff
@@ -0,0 +1,21 @@
+domain: EXT
+type: NVIDIA_JETSON
+synthesis:
+  rules:
+  - identifier: hostname
+    name: hostname
+    encodeIdentifierInGUID: true
+    conditions:
+      - attribute: eventType
+        value: jetsonTegrastats
+      - attribute: metricName
+        value: "ram_used"
+    tags:
```

Comment:
These tags are not included in the test data. Could you include them?

Reply:
Hi @javimb Updated the tags in test data